src/Agent.Worker/Release/ContainerFetchEngine/HttpRetryOnTimeoutHandler.cs

```diff
@@ -26,6 +27,9 @@ public HttpRetryOnTimeoutMessageHandler(HttpRetryOnTimeoutOptions retryOptions,
             HttpRequestMessage request,
             CancellationToken cancellationToken)
         {
+#if OS_LINUX
+            request.Version = HttpVersion.Version11;
```

Comment:
not sure about this, check where `HttpRetryOnTimeoutMessageHandler` was added.

Reply:
Looks like this handler is configured independently, and never calls the VSS handler. I'll keep the override in this case.